modules/react-native-chat/Components/Button.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Text, TouchableOpacity, StyleSheet } from 'react-native';
+
+export default ({ title, onPress }) => {
+    return <TouchableOpacity onPress={onPress}>
+    <Text>{title}</Text>
+  </TouchableOpacity>;
+};
```

Comment:
I would suggest that you define the component and then export it, you lose the name of the function by doing this.

Reply:
Saw this in other places, same suggestion applies.